templates/managed-ec2-linux-v2.j2

```diff
@@ -0,0 +1,345 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  Provision EC2 instance, connect to Jumpcloud, and associate with a jumpcloud systems group.
+Parameters:
+  MaintenanceWindow:
```

Comment:
Your intention is to have people set this?  Hmm, it would be hard to imagine that happening.

Reply:
I want to give people a few options and for it to default to the best window I can think of when they don't select one.

Reply:
ditto here.